recipes/mscxyz/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "mscxyz" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 5b9c01bdd2461cb99dd0cc73f8edaf3025d3063b4e6d0f0c66b5eb46c8078695
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - lxml
+    - python >=3.6
+    - termcolor
+    - tmep >=2.0.1
+
+test:
+  imports:
```

Comment:
This looks like the main package's use is through a command-line application called `mscx-manager`. So maybe a good test to have would be to run that?
```yaml
test:
  script:
    - mscx-manager
```

Given that it's listed under `scripts` in `setup.py`, I'm guessing that's going to make this not work as a `noarch: python` recipe.

Reply:
Got it, let me try adding these